src/homework01/Tasks.kt

```diff
@@ -1,13 +1,149 @@
 package homework01
 
-fun binarySearch(array: List<Int>, f: Int): Int = 0
+const val StartLinear = 5
```

Comment:
FYI, in Java & Kotlin constants are written in _CAPS_LOCK_ with_underscores
So, that would be `START_LINEAR`